test/e2e/e2e.go

```diff
@@ -32,12 +32,12 @@ import (
 
 	// tests to run
 	_ "k8s.io/ingress-nginx/test/e2e/annotations"
-	_ "k8s.io/ingress-nginx/test/e2e/defaultbackend"
-	_ "k8s.io/ingress-nginx/test/e2e/lua"
-	_ "k8s.io/ingress-nginx/test/e2e/servicebackend"
-	_ "k8s.io/ingress-nginx/test/e2e/settings"
-	_ "k8s.io/ingress-nginx/test/e2e/ssl"
-	_ "k8s.io/ingress-nginx/test/e2e/status"
+	// _ "k8s.io/ingress-nginx/test/e2e/defaultbackend"
+	// _ "k8s.io/ingress-nginx/test/e2e/lua"
+	// _ "k8s.io/ingress-nginx/test/e2e/servicebackend"
+	// _ "k8s.io/ingress-nginx/test/e2e/settings"
+	// _ "k8s.io/ingress-nginx/test/e2e/ssl"
+	// _ "k8s.io/ingress-nginx/test/e2e/status"
```

Comment:
I assume this change is from your testing.

Reply:
@clandry94 you can use `FOCUS="Annotations\s-\scanary" make e2e-test` to avoid commenting the code.